### ISSUE TITLE
Bleeding Infusion Still Broken on Blood Kinetiscist 

### ISSUE THREAD

**Issue by mccabefliam** [Truinto/DarkCodex#121]
Always deals 1 damage no matter how many dice are used with blast and can not stack beyond 1. I believe the fix you implemented fixed Wrack but I got to test it more.

**Comment by Truinto** [comment]
Yeah, I know. I have very little time recently and my few attempts to fix it went nowhere. I will fix it eventually. Sorry if this is inconvenient to you.
